implementations/micrometer-registry-stackdriver/src/main/java/io/micrometer/stackdriver/StackdriverMeterRegistry.java

```diff
@@ -138,11 +138,21 @@ public void start(ThreadFactory threadFactory) {
 
     @Override
     public void stop() {
-        if (client != null)
-            client.shutdownNow();
```

Comment:
Removing this from here will potentially be problematic (a resource leak) if the registry is stopped and started multiple times. If it is only stopped as part of `close` then I think this fix works. We should have some kind of tests for this, ideally.